src/SDKs/HDInsight/Management.HDInsight/Properties/AssemblyInfo.cs

```diff
@@ -21,8 +21,8 @@
 
 [assembly: AssemblyTitle("Microsoft Azure HDInsight Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure HDInsight.")]
-[assembly: AssemblyVersion("3.0.0.0")]
-[assembly: AssemblyFileVersion("3.0.0.0")]
+[assembly: AssemblyVersion("3.0.1.0")]
```

Comment:
@wawon-msft please do not update AssemblyVersion except you are bumping major version.
There are very specific cases where you update AssemblyVersion minor/build numbers and this change is not one of them.
Pelase revert your AssemblyVerison change.

Reply:
I've reverted the change.